src/addScore.js

```diff
@@ -0,0 +1,19 @@
+import { saveToApi } from './api.js';
+
+const addScore = () => {
+  const addForm = document.querySelector('.add-form');
+  const name = document.getElementById('name');
+  const score = document.getElementById('score');
+
+  addForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+    saveToApi(name.value, score.value)
+      .then((response) => {
+        if (response.result === 'Leaderboard scores are well created.') {
+          window.reload();
+        }
+      });
+  });
+};
```

Comment:
- [ ] Your submit button is not working, when I submit a new score the page reloads but nothing else happens, kindly fix this issue 🤔